mica/stats/acq_stats.py

```diff
@@ -441,6 +448,43 @@ def _get_obsids_to_update(check_missing=False):
     return obsids
 
 
+def warn_on_acq_anom(acq, starcheck, to_addr):
```

Comment:
Every new "significant" function should have a docstring that describes what it does and the keyword inputs.  Maybe some teeny functions which are completely obvious can have a pass, but as reviewer I spent too much time figuring out just what is `acq`.